### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
Update to our pull request template to replace the `Testing Done` section with `QA Checklist`

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -17,14 +17,23 @@ https://{{head.ref}}--60f9b557105290003b387cd5.chromatic.com
 ## Description
 Closes <ticket>
 
-## Testing done
-
+## QA Checklist
+- [ ] Component maitains 1:1 parity with design mocks
```
Comment (Andrew565) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: ^maintains

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -17,14 +17,23 @@ https://{{head.ref}}--60f9b557105290003b387cd5.chromatic.com
...
+- [ ] Text is consistent with what's been provided in the mocks
+- [ ] Component behaves as expected across breakpoints
+- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
+- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
+- [ ] Tab order and focus state work as expected
+- [ ] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
+- [ ] New components are covered by e2e tests / updates to existing components are covered by existing test suite
```
Comment (Andrew565) [inline .github/PULL_REQUEST_TEMPLATE.md:28]: ^e2e tests; updates (recommend semi-colon for greater clarity; the '/' can be read as "e2e tests OR updates")

**Review by Andrew565 — CHANGES_REQUESTED** [review]
Grammar and spelling typo fix suggestions.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Andrew565) [reply .github/PULL_REQUEST_TEMPLATE.md:28]: This one is a personal nit-pick, do with it what you want. ;)

Comment (micahchiang) [reply .github/PULL_REQUEST_TEMPLATE.md:28]: nah you're right, OR is better here.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -17,14 +17,23 @@ https://{{head.ref}}--60f9b557105290003b387cd5.chromatic.com
 ## Description
 Closes <ticket>
 
-## Testing done
-
+## QA Checklist
```
Comment (jamigibbs) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: Is this checklist for the reviewer of the PR or for the person creating the PR? Or maybe both?

Comment (micahchiang) [reply .github/PULL_REQUEST_TEMPLATE.md:20]: In my head the list should be used by both parties to ensure everything is being met properly.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -17,14 +17,23 @@ https://{{head.ref}}--60f9b557105290003b387cd5.chromatic.com
 ## Description
 Closes <ticket>
 
-## Testing done
```
Comment (jamigibbs) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: This is being mentioned in the [Configuring this pull request](https://github.com/department-of-veterans-affairs/component-library/blob/6b62b3e7d3f4eebcee5ceb3fe64bb7d46531c279/.github/PULL_REQUEST_TEMPLATE.md#configuring-this-pull-request) section (the second checkbox). So we probably need to remove that line now with such a robust QA checklist below now.

**Review by jamigibbs — APPROVED** [review]
Nice addition. Thanks!

**Review by Andrew565 — APPROVED** [review]

**Pull request merged**